data/man/tilix

```diff
@@ -21,28 +21,28 @@ Show the version of Tilix as well as the versions of dependent components.
 Set the working directory of the terminal.
 .TP
 .B \-p \-\-profile=PROFILE_NAME
-Set the starting profile using the name of one of the existing profiles. If a name is passed that does not correspond to an existing profile, the default profile will be used.
+Set the starting profile using the name of one of the existing profiles. If it matches none of those, the default profile will be used.
 .TP
 .B \-t \-\-title=TITLE
 Set the title of the new terminal.
 .TP
 .B \-s \-\-session=SESSION_NAME
-Open the specified session file which has been previously saved. Session files are saved from within Tilix, they store the complete layout of the session enabling the user to re-load it when required. Note that only the layout and layout options are saved, this will not restore the terminal to where it was left off, rather new terminals are created.  
+Specify a previously saved session to open. They store the complete layout of the session, saved and re-loaded from within Tilix when required. Note that only the layout and layout options are saved, this will not restore the terminal to where it was left off, rather new terminals are created.  
```

Comment:
Who are "they"?

Reply:
How about:
"Specify a previously saved session to open and re-load when required. Only the layout and layout options are saved"


"this will not restore the terminal to where it was left off, rather new terminals are created."
This part i don't understand.

Reply:
> Specify a previously saved session to open and re-load when required.

"when required" is not needed - once you've specified a previously saved session, it will be opened and reloaded.

> This part i don't understand.

This part means that the terminals themselves with their states, such as clients running or scrollback history, are not saved, in fact. Only the layout is saved, and based on that new terminals are created.

Reply:
OK, how about:
"Specify a previously saved session to re-load the layout and layout options from. This will not restore the terminal to where it was left off, rather new terminals are created."

Reply:
This sounds right.

However, I'm not sure what was the reason to change the existing text in the first place, but I guess I'll let @gnunn1 decide on that.